src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/compat/patchouli/PatchouliCompat.java

```diff
@@ -291,6 +299,14 @@ public void init(FMLCommonSetupEvent event) {
         ).setSymmetrical(true));
     }
 
+    @NotNull
```

Comment:
Unnecessary NotNull, we assume non-nullability unless explicitly stated (if not, add the package-info for that package).